azure-kusto-data/source/client.js

```diff
@@ -144,6 +144,10 @@ module.exports = class KustoClient {
     _getClientTimeout(endpoint, properties) {
         let timeout = null;
         if (properties != null) {
+            if (properties instanceof ClientRequestProperties && properties.getClientTimeout()){
```

Comment:
Minor:
I know it's just a getter, but as I'm not sure how Node deals with it: should we call getClientTimeout() once?